DCode.Services/Common/CommonService.cs

```diff
@@ -502,6 +506,13 @@ public IEnumerable<Offering> GetOfferings()
             return _offeringModelFactory.CreateModelList<Offering>(offerings);
         }
 
+        public decimal? GetApprovedApplicantHours()
+        {
+            var applicants = _approvedApplicantRepository.GetApprovedApplicants();
+            var hoursWorked = _approvedApplicantModelFactory.CreateModelList<ApprovedApplicant>(applicants).Where(x => x.APPLICANT_ID == 649);

```

Comment:
there is HardCoding here 
Is this intentional?

Reply:
@priyankalekkala :: Should we not get the total hours of the current user alone? Please refer to my comment as well for `IApprovedApplicantRepository `.